nucypher/cli/actions/select.py

```diff
@@ -289,3 +291,18 @@ def select_config_file(emitter: StdoutEmitter,
         config_file = config_files[0]
 
     return config_file
+
+
+def select_card(emitter, card_identifier: str = None, nickname: str = None) -> Card:
```

Comment:
do we need `nickname` as a parameter here? i.e. can we just use the `card_identifier` parameter whose value can either be the card id or nickname.